src/Markdig/Syntax/ContainerBlock.cs

```diff
@@ -234,14 +236,14 @@ public void RemoveAt(int index)
 
         public void Sort(IComparer<Block> comparer)
         {
-            if (comparer == null) ThrowHelper.ArgumentNullException(nameof(comparer));
-            Array.Sort(children, 0, Count, comparer);
+            if (comparer is null) ThrowHelper.ArgumentNullException(nameof(comparer));
+            Array.Sort(children, 0, Count, comparer!);
         }
 
         public void Sort(Comparison<Block> comparison)
         {
             if (comparison == null) ThrowHelper.ArgumentNullException(nameof(comparison));
-            Array.Sort(children, 0, Count, new BlockComparer(comparison));
+            Array.Sort(children, 0, Count, new BlockComparer(comparison)!);
```

Comment:
This should be needed right after the `new`

Reply:
I backed up out making the comparison delegate nullable, so these suppressions are no longer needed.